sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/converters/OptionsConverter.java

```diff
@@ -3,26 +3,8 @@
 
 package com.azure.digitaltwins.core.implementation.converters;
 
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinModelsListOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsAddOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsAddRelationshipOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsDeleteOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsDeleteRelationshipOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsUpdateComponentOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsUpdateOptions;
-import com.azure.digitaltwins.core.implementation.models.DigitalTwinsUpdateRelationshipOptions;
-import com.azure.digitaltwins.core.implementation.models.EventRoutesListOptions;
-import com.azure.digitaltwins.core.implementation.models.QueryTwinsOptions;
-import com.azure.digitaltwins.core.models.CreateOrReplaceDigitalTwinOptions;
-import com.azure.digitaltwins.core.models.CreateOrReplaceRelationshipOptions;
-import com.azure.digitaltwins.core.models.DeleteDigitalTwinOptions;
-import com.azure.digitaltwins.core.models.DeleteRelationshipOptions;
-import com.azure.digitaltwins.core.models.ListDigitalTwinsEventRoutesOptions;
-import com.azure.digitaltwins.core.models.ListModelsOptions;
-import com.azure.digitaltwins.core.models.QueryOptions;
-import com.azure.digitaltwins.core.models.UpdateComponentOptions;
-import com.azure.digitaltwins.core.models.UpdateDigitalTwinOptions;
-import com.azure.digitaltwins.core.models.UpdateRelationshipOptions;
+import com.azure.digitaltwins.core.implementation.models.*;
```

Comment:
Nit: while this doesn't have a runtime performance impact, it does have a small compiler performance impact to use start / * import rather than one-by-one. I would prefer the latter to make it very explicit what classes we are importing from what namespace.